mission-control-client/src/components/dashboard/user-dashboard/DashboardBanner.js

```diff
@@ -1,13 +1,14 @@
-import React from "react";
+import React, { useEffect } from "react";
 import { connect } from "react-redux";
 
 const DashboardBanner = props => {
+
   const programs = ["web", "ux/ui", "ds"];
 
   return (
     <div className="dashboard-banner-container">
       <div className="dashboard-banner-head">
-        <p>{props.activeProductStore.active ? props.activeProductStore.active.name : "Loading..."}</p>
+        <p>{props.activeProjectStore.active ? props.activeProjectStore.active.project.project.name : "Loading..."}</p>
```

Comment:
`project.project` is questionable. Why can't it be just `props.activeProductStore.active.name`?

Reply:
In our code cleanup we can for sure change this